Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -973,16 +973,6 @@ private void ReleaseRigidBody(Vector3 velocity, Vector3 angularVelocity)
             {
                 rigidBody.useGravity = wasGravity;
                 rigidBody.isKinematic = wasKinematic;
-
-                if (releaseBehavior.HasFlag(ReleaseBehaviorType.KeepVelocity))
```

Comment:
Are these flags used elsewhere anymore, or do we basically ignoring them now?

Reply:
yeah they're only used here

I can get rid of the variable if we decide to go with option 3's behavior

Reply:
See comment in the main thread